### PR TITLE
Redesign projects list three rows of three

### LINKED ISSUE #797: Projects index page redesign
Imported from https://www.pivotaltracker.com/story/show/79660056

**Story**
As an agile ventures user
So that I can (find a new | create a new | find my | promote my) project(s)
I would like to see a fully featured projects index view

**Supporting documents**
Feature request on AG: http://agileventures.org/projects/websiteone/documents/project-index-page-ux-proposal
Direct link to wireframes: https://drive.google.com/file/d/0B9FqKjpP8_MUQXBlcWZ2R3JzV2c/edit?usp=sharing

**Tasks:**
- No tab for current users projects
- No search
- No featured project

### PR DESCRIPTION
**Pull request by mattwr18**
fixes #797 

Add hover effect
Get tests passing

### CODE REVIEW

**Comment by tansaku** [comment]
looks okay superficially - haven't been able to give detailed review yet

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -279,6 +279,10 @@ def current_fullpath
   page.find(:css, '#sidebar')
 end
 
+Then(/^I should not see the sidebar$/) do
+  expect(page).to_not have_selector('#sidebar')
```
Comment (roschaefer) [inline features/step_definitions/basic_steps.rb]: @mattwr18 same same, don't assert sth. is not visible. At least check sth. that's there for sure and compare e.g. the html structure :man_shrugging: or maybe even consider not to write tests. Those tests are likely to provide little confidence but will give some extra work on refactorings.

File: features/sidebar.feature [hunk 2]
```diff
@@ -53,5 +53,3 @@ Feature: Sidebar navigation
     Given I am on the "Show" page for project "hello mars"
     Then I should see "hello moon" before "hello pluto"
     And I should see "hello pluto" before "hello world"
-
-
```
Comment (roschaefer) [inline features/sidebar.feature]: This whole feature :point_up: is particularly verbose. Reading the feature description and especially the scenario, it is not clear to me what the problem actually is. Don't write cucumber scenarios in a way that let's you re-use existing step definitions. 1) write your feature 2) run `bundle exec cucumber` 3) copy-paste the generated step definitions and implement them. It is OK to have many step defintions.

See this blog post for details: http://aslakhellesoy.com/post/11055981222/the-training-wheels-came-off
Aslak Hellesøy is the inventor of cucumber-ruby and he gives advice on how not to write boring scenarios. :wink:

Comment (roschaefer) [reply features/sidebar.feature]: and there is more: https://cucumber.io/blog/2016/07/01/cucumber-antipatterns-part-one

Comment (tansaku) [reply features/sidebar.feature]: definite need for refactoring - well flagged @roschaefer 
@mattwr18 - yes definitely a great thing for a refactoring ticket (when you didn't write the original tests) - did you create a refactoring ticket already?

**Comment by tansaku** [comment]
great work @mattwr18 and @javpet - is this ready to merge now?

**Comment by mattwr18** [comment]
this is ready @tansaku, we talked about the benefits and downsides to infiniteScroll, but that can be a separate conversation and ticket at a later date. big thanks to @javpet for making this actually look good enough to merge in!

**Comment by tansaku** [comment]
great work @javpet and @mattwr18 

now to get it into production :-)

**Pull request merged**